docs/src/examples/do-events.jl

```diff
@@ -0,0 +1,244 @@
+#=
+# Dansgaard-Oescher events
+
+The $\delta^{18}O$ timeseries of the North Greenland Ice Core Project ([NGRIP](https://en.wikipedia.org/wiki/North_Greenland_Ice_Core_Project)) are, to this date, the best proxy record for the Dansgaard-Oeschger events ([DO-events](https://en.wikipedia.org/wiki/Dansgaard%E2%80%93Oeschger_event)). DO-events are sudden warming episodes of the North Atlantic, reaching 10 degrees of regional warming within 100 years. They happened quasi-periodically over the last glacial cycle due to transitions between strong and weak states of the Atlantic Meridional Overturning Circulation and might be therefore be the most prominent examples of abrupt transitions in the field of climate science. We here propose to hindcast these events by applying the theory of Critical Slowing Down (CSD) on the NGRIP data, which can be found [here](https://www.iceandclimate.nbi.ku.dk/data/) in its raw format. This analysis has already been done in [boers-early-warning-2018](@cite) and we here try to reproduce Figure 2.d-f.
+
+## Preprocessing NGRIP
+
+Over this example, it will appear that the convenience of TransitionsInTimeseries leads the bulk of the code to be written for plotting and preprocessing. The latter consists in various steps $i = \lbrace 1,2,3 \rbrace$:
```

Comment:
Here say something like "data pre-processing is not part of TransitionsInTimeseries.jl, but a step the user has to do before using the package. To present an example representing a complete scientific workflow, we will showcase typical data pre-processing here, that consist of the following steps:"

Then, after the numbered list, say "coincidentally, we will see how much convenience TransitionsInTimeseries.jl brings to the table. Almost all code in this example is dedicated to pre-processing, not to estimating transitions."

p.s.: Either use `TransitionsInTimeseries` (code font) or TransitionsInTimeseries.jl (with .jl).

Reply:
Added the first suggestion, second one was already in there. Using TransitionsInTimeseries.jl.